code/modules/plumbing/plumbers/reaction_chamber.dm

```diff
@@ -1,6 +1,7 @@
 ///a reaction chamber for plumbing. pretty much everything can react, but this one keeps the reagents separated and only reacts under your given terms
+//base class is a simpler reaction chamber for bartenders
```

Comment:
This will break the doc comment. You need three '/'s. Your comment also does not describe what the actual machine does